src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        Random random = new Random();
+        int number = random.nextInt(100);
+        ColorSupplier col = new ColorSupplier();
+        Ball ball = new Ball();
+        ball.setValue(number);
+        ball.setColor(col.getRandomColor());
+        return ball;
+    }

```

Comment:
```suggestion
public class Lottery {
    private final Random random = new Random();
    private final ColorSupplier colorSupplier = new ColorSupplier();

    public Ball getRandomBall() {
        int number = random.nextInt(100);
        Ball ball = new Ball();
        ball.setValue(number);
        ball.setColor(col.getRandomColor());
        return ball;
    }

```
ColorSupplier and Random must be a class variable